posts/2019-07-15-Rust-1.37.0.md

```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: "Announcing Rust 1.37.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.37.0. Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.37.0 is as easy as:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the appropriate page on our website, and check out the [detailed release notes for 1.37.0][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1370-2019-08-15
+
+## What's in 1.37.0 stable
+
+The highlights of Rust 1.37.0 include referring to `enum` variants through `type` aliases, built-in `cargo vendor`, unnamed `const` items, profile-guided optimization, a `default-run` key in Cargo, and `#[repr(align(N))]` on `enum`s. Read on for a few highlights, or see the [detailed release notes][notes] for additional information.
+
+### Referring to `enum` variants through `type` aliases
+
+With Rust 1.37.0, you can now refer to `enum` variants through type aliases. For example:
+
+```rust
+type ByteOption = Option<u8>;
+
+fn increment_or_zero(x: ByteOption) -> u8 {
+    match x {
+        ByteOption::Some(y) => y + 1,
+        ByteOption::None => 0,
+    }
+}
+```
+
+In implementations, `Self` acts like a type alias. So in Rust 1.37.0, you can also refer to `enum` variants with `Self::Variant`:
+
+```rust
+impl Coin {
+    fn value_in_cents(&self) -> u8 {
+        match self {
+            Self::Penny => 1,
+            Self::Nickel => 5,
+            Self::Dime => 10,
+            Self::Quarter => 25,
+        }
+    }
+}
+```
+
+[type_rel_report]: https://github.com/rust-lang/rust/pull/61682/#issuecomment-502472847
+
+To be more exact, Rust now allows you to refer to `enum` variants through *"type-relative resolution"*, `<MyType<..>>::Variant`. More details are available in [the stabilization report][type_rel_report].
+
+### Built-in Cargo support for vendored dependencies
+
+[vendor-crate]: https://crates.io/crates/cargo-vendor
+
+After being available [as a separate crate][vendor-crate] for years, the `cargo vendor` command is now integrated directly into Cargo. The command fetches all your project's dependencies unpacking them into the `vendor/` directory, and shows the configuration snippet required to use the vendored code during builds.
+
+There are multiple cases where `cargo vendor` is already used in production: the Rust compiler `rustc` uses it to ship all its dependencies in release tarballs, and projects with monorepos use it to commit the dependencies' code in source control.
```

Comment:
```suggestion
There are multiple cases where `cargo vendor` is already used in production: The Rust compiler `rustc` uses it to ship all its dependencies in release tarballs, and projects with monorepos use it to commit the dependencies' code in source control.
```

Reply:
It's not a new sentence, so why capitalize? (If you used `.` as punctuation I'd be inclined to agree.)

Reply:
Ah yeah, also fine by me.